[Docs] Fix information about activation and deactivation of chainable controllers (#1104)
ros-controls/ros2_control: controller_manager/doc/controller_chaining.rst

This is the same process as done by ``ResourceManager`` and hardware interfaces.
Controller manager maintains "claimed" status of interface in a vector (the same as done in ``ResourceManager``).

Activation and Deactivation Chained Controllers
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

> The chained controller must be activated and deactivated together or in the proper order.

Reviewer: ```suggestion
Chained controllers must be activated and deactivated together or in the proper order.
```

This means you must first activate all following controllers to have the preceding one activated.
For the deactivation there is the inverse rule - all preceding controllers have to be deactivated before the following controller is deactivated.

> If you imagine a chain of controllers, you can not add a chain link or break the chain in the middle.

Reviewer: ```suggestion
One can also think of it as an actual chain, you can not add a chain link or break the chain in the middle.
```
